local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -99,6 +104,18 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
+// fullPath is the pathname of any file within the mounted filesystem. Capacity
+// returned here is total capacity available to non-root users, and does not include
+// fs reserved space.
+func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
```

Comment:
yeah, `usage + available` is basically the idea.

@ianchakeres available in fs.go means remaining bytes available to be consumed, while available here means capacity that can be used by normal users. so we can't change the definition of available in fs.go.  It may worth taking a look if returning a new field for availablebytes is necessary though, but i don't know if there is use case. Even if there is, it can be calculated by just using usage+available. 

But you raise another IMPORTANT problem, that we should change to a better name......